### ISSUE TITLE
Bug: Make linux install script compliant

### ISSUE THREAD

**Issue by Mycobee** [git-ecosystem/git-credential-manager#1468]
The linux-install-source script throws an error because of the lack of space between the bracket and bang

```
$ ./linux-install-source.sh
./linux-install-source.sh: 29: [!: not found
```

**Comment by Mycobee** [comment]
Here's the fix
https://github.com/git-ecosystem/git-credential-manager/pull/1469

**Comment by mjcheetham** [comment]
Thank you for spotting this issue, and the fix in #1469!

This has now been merged to `main`.
